tests/roles/mariadb_copy/templates/pre_checks.bash

```diff
@@ -5,7 +5,7 @@
 {{ mariadb_copy_shell_vars_dst }}
 
 # Test connection to podified DBs (show databases)
-oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
+oc run mariadb-client -n {{ rhoso_namespace }} ${MARIADB_CLIENT_ANNOTATIONS} --image $MARIADB_IMAGE -i --rm --restart=Never -- \
     mysql -rsh "$PODIFIED_MARIADB_IP" -uroot -p"$PODIFIED_DB_ROOT_PASSWORD" -e 'SHOW databases;'
-oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
+oc run mariadb-client -n {{ rhoso_namespace }} ${MARIADB_CLIENT_ANNOTATIONS} --image $MARIADB_IMAGE -i --rm --restart=Never -- \
```

Comment:
Can you also fix this in the docs? Can be proposed in another PR so that we don't lose the CI results here.

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc?plain=1#L148-L151